pydoctor/sphinx_ext/__init__.py

```diff
@@ -1,3 +1,51 @@
 """
 Public and private extensions for Sphinx.
 """
+import os
+import subprocess
+
+
+def get_source_reference() -> str:
+    """
+    Return the reference for current build to be used when linking
+    the source files.
```

Comment:
```suggestion
    Return a reference to the particular source version used in this build.
    This can be used in the URL passed with C{--html-viewsource-base}, to link to the
    version of the source code that corresponds with the generated documentation.
```